src/core/core.tooltip.js

```diff
@@ -40,6 +40,9 @@ module.exports = function(Chart) {
 		displayColors: true,
 		callbacks: {
 			// Args are: (tooltipItems, data)
+			tooltipOpened: helpers.noop,
```

Comment:
I'd hope so... my plugin seemed to be notified. Do I need to do something additional here? I just followed the structure of the other :)